t/Hydra/Controller/Job/builds.t

```diff
@@ -0,0 +1,31 @@
+use strict;
+use warnings;
+use Setup;
+use Test2::V0;
+use Catalyst::Test ();
+use HTTP::Request::Common;
+
+my $ctx = test_context();
+
+Catalyst::Test->import('Hydra');
+
+my $builds = $ctx->makeAndEvaluateJobset(
+    expression => "basic.nix",
+    build => 1
+);
+
+my $build = $builds->{"empty_dir"};
+my $jobset = $build->jobset;
+my $project = $build->project;
+
+subtest "/job/PROJECt/JOBSET/JOB/all" => sub {
```

Comment:
Minor nit: any reason why the final `t` in `PROJECt` is lower-case?

Reply:
Typo + copypasta I'm sure!